proxy.go

```diff
@@ -775,6 +775,32 @@ func (p *proxyObject) ownKeys(all bool, _ []Value) []Value { // we can assume ac
 	return p.target.self.ownKeys(all, nil)
 }
 
+func (p *proxyObject) ownEntries(all bool, _ []Value) []Value { // we can assume accum is empty
+	var keys []Value
+	var ok bool
+	if keys, ok = p.proxyOwnKeys(); !ok {
+		keys = p.target.self.ownKeys(all, nil)
+	}
+
+	entries := make([]Value, 0, len(keys))
+	for _, k := range keys {
+		desc, ok := p.proxyGetOwnPropertyDescriptor(k)
+		if !ok {
+			desc = propToValueProp(p.target.getOwnProp(k))
```

Comment:
No test is actually testing this, so I should probably try to figure out way to test it, as I am not certain this is the correct way to do it either way